omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -8,6 +8,15 @@
     Id="*" is to enable upgrading. * means that the product ID will be autogenerated on each build.
     Name is made of localized product name and version number.
   -->
+  <?if $(var.Platform) = x64 ?>
+    <?define ProductName = "Product Name (64 bit)" ?>
```

Comment:
should we not have 64bits as a default and 32bit as exception ? Product name would be `Product Name` for x64 and "Product Name (32 bit)" for 32bit

Reply:
oof.  Good catch. Copy & paste error.  Product Name is defined below, and will be the same regardless of 32/64 bitness